### PR TITLE
Brain Dataset Support

### PR DESCRIPTION
**Pull request by tullie**
This PR adds support for the upcoming brain dataset. Specifically it adds:
- Handling of arbitrary sized kspace and target images (in dataloaders and UNet model)
- Adding equispaced masks, which can be chosen with a command line argument
- Hard coding best regularization weights for each brain sequence type when using TV.

### CODE REVIEW

File: common/subsample.py [hunk 1]
```diff
@@ -9,9 +9,18 @@
 import torch
 
 
-class MaskFunc:
+def mask_for_mask_type(mask_type_str, center_fractions, accelerations):
```
Comment (anuroopsriram) [inline common/subsample.py]: Rename this function to `create_mask_from_type(...)`.

File: models/cs/run_bart_val.py [hunk 2]
```diff
@@ -18,7 +18,7 @@
 import bart
```
Comment (anuroopsriram) [inline models/cs/run_bart_val.py]: Update the [models/cs/README.md](https://github.com/facebookresearch/fastMRI/blob/master/models/cs/README.md) file explaining how the run commands differ for brain MRIs and knee MRIs (`--mask-type equispaced` for brains and `--mask-type random` for knees).

**Comment by anuroopsriram** [comment]
Added some inline comments. Incorporate them as you see fit.

Additional general comments:
1. Update the [models/cs/README.md](https://github.com/facebookresearch/fastMRI/blob/master/models/cs/README.md) file explaining how the run commands differ for brain MRIs and knee MRIs (--mask-type equispaced for brains and --mask-type random for knees).

2. Similarly, update the [models/unet/README.md](https://github.com/facebookresearch/fastMRI/tree/master/models/unet) and [models/zero_filled/README.md](https://github.com/facebookresearch/fastMRI/tree/master/models/zero_filled)

3. Since you renamed `MaskFunc` to `RandomMaskFunc`, please also change all usages of `MaskFunc` accordingly. For example, [fastMRI_tutorial.ipynb](https://github.com/facebookresearch/fastMRI/blob/master/fastMRI_tutorial.ipynb) and [data/README.md](https://github.com/facebookresearch/fastMRI/blob/master/data/README.md).

**Comment by tullie** [comment]
Going to go ahead and merge because I want this in before the blog post tomorrow. Feel free to comment more and i'll fix it up later if needed.
